README.md

```diff
@@ -1,3 +1,12 @@
+# Build Status
+[![Build Status](https://github.com/compucorp/uk.co.compucorp.civicase/workflows/Tests/badge.svg)](https://github.com/compucorp/uk.co.compucorp.civicase/workflows/Tests/badge.svg)
+
+*Dependent Civicrm Extentions Build Status*
```

Comment:
s/Extentions/Extensions

Not sure if we really need to have build status for the dependent extension build status here tbh since we already added this on each extension repo. If the build for a particular dependent extension fail, it might totally be unrelated to civicase or what do you think?